consensus/bor/api.go

```diff
@@ -105,3 +111,41 @@ func (api *API) GetCurrentValidators() ([]*Validator, error) {
 	}
 	return snap.ValidatorSet.Validators, nil
 }
+
+// GetRootHash gets the current validators
+func (api *API) GetRootHash(start uint64, end uint64) ([]byte, error) {
+	blockHeaders := make([]*types.Header, end-start+1)
+	wg := new(sync.WaitGroup)
+	// do we want to limit the # of concurrent go routines?
```

Comment:
Yes. That would be great. 10 goroutines would be sufficient.

Reply:
Limited to 20 goRoutes.